TelepresenceBot/server-unit/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "tpbot_server",
+  "version": "0.1.0",
+  "description": "Middleman server for the tpbot android application.",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/novoda/spikes/tree/tpbot-feature-branch"
```

Comment:
Maybe we could just put the repo URL already, as we hopefully will merge soon to master